primo-zip.py

```diff
@@ -25,13 +26,14 @@ def swap_secrets_and_values(secret_data, types_to_check, directory, secret_visib
                     secret_value = str(
```

Comment:
@ztucker4 I can't target most unchanged lines via the PR review interface, but I think there's an opportunity to improve the code starting with `cnt = 0` , lines 22-27.

The current code uses the artificial numerical indexes, `cnt` and `i`, to iterate through `secret_data['secret_details']`.  I suspect this could be improved with something like:
```
for secret in secret_data['secret_details']:
    secret_name = secret['secret_name']
    secret_value = secret['secret_value']
```
I dropped the `str` as the json only contains string values.

That eliminates the need for `cnt`, I think, and `i` isn't used other than to drive the iteration through the secrets.  So this should work, and is more "Pythonic".